bin/create-probot-app.js

```diff
@@ -145,14 +146,34 @@ inquirer.prompt(prompts)
   })
   .then(results => {
     results.forEach(fileinfo => {
+      if (fileinfo.skipped === false &&
+        path.basename(fileinfo.path) === 'gitignore'
```

Comment:
Edge case: Because create-probot-app is idempotent, if a file is named gitignore in the directory of the initializing Probot app, no .gitignore file will be created.